.github/workflows/main.yml

```diff
@@ -7,23 +7,7 @@ on:
     branches: [ master ]
 
 jobs:
-  test-package-plugin-macos-12:
```

Comment:
The macOS 12 runner is now deprecated so needs to be removed:

<img width="1014" alt="image" src="https://github.com/user-attachments/assets/d82b2da5-d797-4ca6-b644-6f16c4268f5b">